.github/ISSUE_TEMPLATE/05_onboard.yml

```diff
@@ -18,7 +18,7 @@ body:
         - label: Given a tour of Github from a Core member
         - label: Understand how to Create Issues and do a Pull Request
         - label: Understand the Programming Strategy
-        - label: Read and understand [Developer Guides Articles](https://pharmaverse.github.io/admiral/index.html)
+        - label: Read and understand [Developer Guides Articles](https://pharmaverse.github.io/admiral/cran-release/index.html)
```

Comment:
The developer guides were moved to admiraldev. Thus it should link to https://pharmaverse.github.io/admiraldev/devel/articles/index.html.